draft-ietf-taps-impl.md

```diff
@@ -920,6 +920,14 @@ Close:
 Abort:
 : Calling `Abort` on a TCP Connection indicates that the Connection should be immediately closed by sending a RST to the peer (ABORT.TCP).
 
+CloseGroup:
+: Calling `CloseGroup` on a TCP Connection (CLOSE.TCP) is identical to calling `Close` on all TCP Connections
+within a group.
```

Comment:
How about:

```suggestion
: Calling `CloseGroup` on a TCP Connection (CLOSE.TCP) is identical to calling `Close` on this Connection and on all Connections in the same ConnectionGroup.
```
This is essentially what it says in Section 10 in the Interface draft, and perhaps the clearest way to phrase this.

Reply:
I like this - I think it should be phrased using that style for all the calls below. "within a group" is just a touch too vague IMO... it needs to be the group this connection is in, even though this is super obvious.

Reply:
Applied, thanks!